docs/source-2.0/additional-specs/protocols/smithy-rpc-v2.rst

```diff
@@ -214,10 +214,16 @@ Buffered RPC requests for the ``rpcv2Cbor`` protocol SHOULD include a
 ``Content-Length`` header. Event streaming requests MUST NOT specify a content
 length (instead using ``Transfer-Encoding: chunked`` on HTTP/1.1).
 
-Requests with event stream responses for the ``rpcv2Cbor`` protocol MUST
-include an ``Accept`` header set to the value
-``application/vnd.amazon.eventstream``. Other forms of content streaming MAY be
-added in the future, utilizing different values for ``Accept``.
+Requests for the ``rpcv2Cbor`` protocol MUST use the following behavior for
+setting an ``Accept`` header:
+
+* Buffered RPC requests: the value of the ``Accept`` header MUST be
```

Comment:
I don't have a better proposal (maybe "non-event stream requests", or "request-response requests"), but using "buffered" to make the distinction here is confusing to me.

Reply:
This is the same wording and layout as is used for the `Content-Type` header above.